content/03.text-mining.md

```diff
@@ -18,8 +18,43 @@
 
 #### Unsupervised Machine Learning
 
-1. Mention Clustering Analysis
-2. Mention Co-Occurrence approaches
+Unsupervised machine learning consists of drawing inferences within data without the use of labels.
+Common methods of unsupervised learning involve some form of clustering or statistical calculations.
+In this section we discuss methods that use unsupervised learning to detect relationship asserting sentences from text.
+
+Majority of unsupervised text mining methods use some form of co-occurrence (Table {@tbl:unsupervised-methods-text-mining}), which is an instance of two entities appearing together in text.
+Two popular databases DISEASES [@doi:10.1016/j.ymeth.2014.11.020] and STRING [@doi:10.1093/nar/gku1003] were populated using a co-occurrence scoring method on PubMed abstracts.
+This scoring method measured the frequency of co-mention pairs both within individual sentences and within the abstracts itself.
+Mention pairs that occur more than chance were assumed to have a high likelihood of association or interaction.
+This approach was able to recall a significant amount of relationships, but had a hard time capturing relationships that are rare or underreported.
+ 
+Besides PubMed abstracts, a study that used a similar approach to mine full articles for protein-protein interactions and other protein related information [@doi:10.1371/journal.pcbi.1005962].
```

Comment:
This topic sentence needs to be reworked a bit to highlight the major point of this paragraph. The paragraph itself is really important!